base/paths/paths.go

```diff
@@ -68,8 +70,20 @@ func Subdir(dirs ...string) string {
 	return strings.Join(dirs, "/")
 }
 
+func findGomacroDir() string {
+	pkg := filepath.Join("github.com", "cosmos72", "gomacro") // vendored copies of gomacro may need to change this
```

Comment:
Wouldn't it be better to make this `pkg` variable a parameter to the function?

It would be simpler for people with a vendored directory to reuse this code.

Reply:
Sure. Something like:

```
GoPkg      = filepath.Join("github.com", "cosmos72", "gomacro") // vendored copies of gomacro may need to change this
GoSrcDir   = Subdir(filepath.SplitList(build.Default.GOPATH)[0], "src")
GomacroDir = findGomacroDir(GoPkg)
```

?